src/EditorFeatures/CSharpTest/Completion/CompletionProviders/DeclarationNameCompletionProviderTests.cs

```diff
@@ -25,6 +25,185 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Completion.CompletionSe
 {
     public class DeclarationNameCompletionProviderTests : AbstractCSharpCompletionProviderTests
     {
+        internal const string Span = @"
```

Comment:
Copy/paste from:

https://github.com/dotnet/roslyn/blob/afd10305a37c0ffb2cfb2c2d8446154c68cfa87a/src/Compilers/Test/Utilities/CSharp/TestSources.cs#L11

Reply:
`ReferenceAssemblies.Net.Net50` should already be available to tests, but I'm not sure `AbstractCSharpCompletionProviderTests` is set up to use it. Such a change should probably be a separate pull request.